src/main/java/org/fundacionjala/pivotal/model/pageobjects/AbstractBrowser.java

```diff
@@ -0,0 +1,17 @@
+package org.fundacionjala.pivotal.model.pageobjects;
+
+import org.openqa.selenium.WebDriver;
+
+/**
+ * Parent of Page Objects.
+ * @author Angelica Lopez, Kevin Sanchez - AWT-[01].
+ * @version 0.1
+ */
+public abstract class AbstractBrowser {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Convert the abstract class "AbstractBrowser" into an interface. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1610)